x-pack/plugins/elastic_assistant/server/routes/post_actions_connector_execute.ts

```diff
@@ -107,6 +112,15 @@ export const postActionsConnectorExecuteRoute = (
             };
           }
 
+          const connectorId = decodeURIComponent(request.params.connectorId);
+          const connectors = await actionsClient.getBulk({
+            ids: [connectorId],
+            throwIfSystemAction: false,
+          });
```

Comment:
```suggestion
```
It looks like Patryk brought back some connector code we no longer need because `actionTypeId` is a required body parameter. Please remove and replace the below references to `connectors[0]?.actionTypeId` with the already declared const, `actionTypeId`

Reply:
This also means you can delete the const `actionsClient`